source/databricks/package/constants/partition_naming.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+GLN_VALUE_FOR_GRID_AREA = "grid_area"  # When a result includes the entire grid area we use this text to set the folder name ("gln=grid_area")
```

Comment:
I think this will make intellisense work
```suggestion
GLN_VALUE_FOR_GRID_AREA = "grid_area"
"""When a result includes the entire grid area we use this text to set the folder name ("gln=grid_area")"""
```